script/diff_graphql/bulk.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+# Using a list of base paths, render each page with Content Store and Publishing
+# API's GraphQL endpoint as the data source and diff the results
+
+source script/diff_graphql/functions.sh
+
+diff_style='normal'
+output_dir='tmp/diff_graphql_output'
+help_text="script/diff_graphql/bulk.sh
+
+DESCRIPTION
+     Using a list of base paths, render each page with Content Store and
+     Publishing API's GraphQL endpoint as the data source and diff the results.
+
+ARGUMENTS
+     --base-paths-file-path (required)
+             Path to a file containing a list of base paths (e.g. /world). There
+             should be one base path per line and an empty line at the end of
+             the file.
+
+     --diff-style
+             A style for the diff. Supports three different styles:
+
+             s or side-by-side
+                     A side-by-side view of changed lines. Long lines will be
+                     truncated.
+
+             u or unified
+                     A diff with before/after lines in the same section and a
+                     few lines of context.
+
+             default (any other value)
+                     A standard diff, where for each change the before lines
+                     precede the after lines without any context.
+
+     --environment (required)
+             The environment to which requests should be sent. One of:
+
+             i       Integration
+             s       Staging
+             p       Production
+
+     --output-dir
+             Where to save the diff files. The script will output one diff file
+             per provided base path. By default this is tmp/diff_graphql_output.
+
+     --password (required when environment is integration or staging)
+             The password required to access the publishing service on the given
+             environment.
+
+     --username (required when environment is integration or staging)
+             The username required to access the publishing service on the given
+             environment."
```

Comment:
Should this usage text also be included in `run.sh`?

Reply:
These command-line options aren't supported in `run.sh`

Reply:
Or do you mean the descriptions? It sort of tries to explain things in a more concise way but maybe the more detailed descriptions would be useful